internal/services/nginx/nginx_deployment_resource.go

```diff
@@ -95,12 +97,24 @@ func (m DeploymentResource) Arguments() map[string]*pluginsdk.Schema {
 
 		"location": commonschema.Location(),
 
+		"capacity": {
+			Type:         pluginsdk.TypeInt,
+			Optional:     true,
+			ValidateFunc: validation.IntPositive,
```

Comment:
it would be nice to include a min - max range range for this here if this is documented anywhere?

Reply:
We have server side validation currently for min and max capacity. It has changed a lot over the last few months and even now is a subscription based feature flag based off of enterprise agreements. So the documented min/max is not really the min/max as of now.

This is all to say right now the team has preferred to keep that logic server side until we can really settle into what our min/max will be long term.